java/org/openobservatory/measurement_kit/nettests/HttpInvalidRequestLineTest.java

```diff
@@ -0,0 +1,12 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+package org.openobservatory.measurement_kit.nettests;
+
+public class HttpInvalidRequestLineTest extends OoniTestBase {
+
+    public HttpInvalidRequestLineTest() {
+        super("http_invalid_request_line");
+    }
+}
```

Comment:
@lorenzoPrimi, To define more ooni tests, including web connectivity, I think the way to go is to define more classes like this one that inherit from `OoniTestBase`. (This should answer the question you posed me yesterday.)